src/settings.rs

```diff
@@ -60,7 +59,6 @@ impl kubewarden::settings::Validatable for Settings {
             .rules
```

Comment:
Adding here because it's the only place of the `Settings::validate` where I can put a comment :smile: 

The whole validate has to be rewritten to match what we described on the parent issue:

>  only one directive can be set
>  for more complex logic you can use a Policy Group and
>  combine multiple instances of this policy using the logical
>  operators of your choice

I would also rewrite the `Settings` struct to **not** have a list of rules, but have only one rule. That should be enough for the validation